### ISSUE TITLE
Feature request: Export preview and Independent export

### ISSUE THREAD

**Issue by colomet** [BCcampus/pressbooks-mpdf#37]
### Description 

To be able to see a preview of the PDF page for each chapter or part (by using the pdf settings for such preview).
To be able to export an independent chapter or part from the edit page.

**Expected behaviour** 

To have in each page a buttom where we can see in a pop up how would be the exported current page. 
Also a way to export just one chapter or part without export the full book.

**Actual behaviour** 

Now, if we need to know how looks like a page, we need to export a full book and to take a look. Is important to know how looks like a PDF page, the current way is slow.
Also, some times we just need to give one chapter, to export each chapter as independent would give us more flexibility as in the current way, many settings are necesary for such purpose.

### System Information

- Pressbooks mPDF: [2.0.0.] 
- Pressbooks: [4.5] 
- WordPress: [4.9.1]

**Comment by bdolor** [comment]
> Also a way to export just one chapter or part without export the full book

@colomet — the ability to select which chapters will be exported (one or many, or all) is already a feature of PB. On the `Organize` page in the admin area, unchecking the checkbox for Export will eliminate that particular chapter from the export file.

![image](https://user-images.githubusercontent.com/2048170/33847311-f06838e8-de5e-11e7-8f8a-ee2e41152e4a.png)

**Comment by colomet** [comment]
Yes, but the current process is not flexible, it just works. The current process works perfect for exporting the book, and allowing for the download. 
But if by working in a chapter we need to know if the content fits into the page (some times, because of one line, we need one extra page), we need to export all the book, to go to the export page, after , to delete the book as is not as good as we need.... time consuming. To be able to export one page on the fly, helps for the creation process also for a simple sharing.

Maybe we could also to offer that functionality in the front end for registered users. In that way, students can print one page while they are studing and download it in a perfect stiling.

**Comment by bdolor** [comment]
Okay, thanks for fleshing out that use case a little bit for me.

**Comment by connerbw** [comment]
We worked on preview a bit in the summer but stopped because the implementation was not good enough:

https://github.com/pressbooks/pressbooks/pull/827

The backend code is still in pressbooks. It can be activated by appending ` ?preview[0]=111&preview[1]=222&preview[2]=333` to a URL where 111, 222, and 333 are post_ids.

Examples: 

+ http://pressbooks.test/fixerupper/wp-admin/admin.php?export_formats%5Bpdf%5D=1&page=pb_export&preview%5B0%5D=5&export=yes&_wpnonce=5fb6414a87
+ http://pressbooks.test/fixerupper/wp-admin/admin.php?export_formats%5Bepub%5D=1&page=pb_export&preview%5B0%5D=5&export=yes&_wpnonce=5fb6414a87
+ http://pressbooks.test/fixerupper/wp-admin/admin.php?export_formats%5Bxhtml%5D=1&page=pb_export&preview%5B0%5D=5&export=yes&_wpnonce=5fb6414a87

There's even a GUI in[ this reverted commit](https://github.com/pressbooks/pressbooks/pull/827/commits/d3d9003501e796d245760130aff77c0b237f9041) that may be salvagable:

![preview-buttons](https://user-images.githubusercontent.com/812192/33849447-97daad58-de7e-11e7-85b4-78d4caa8a957.png)

Clicking on any of those buttons would open a new tab previewing only the post you are in. Interbally it was decided that the GUI was a prototype at best so please do not implement it without discussing with us first. 

We would prefer "Preview" be a Pressbooks wide feature, not a MPDF feature. If this somehow becomes part of your sprint I would appreciate you talk to @greatislander to coordinate efforts. We will probably be breaking things moving forward.

**Comment by bdolor** [comment]
I agree that this may be better served as a pressbooks wide feature and not something specific to Mpdf.
